yarn.lock

```diff
@@ -12439,6 +12439,11 @@ lodash.templatesettings@^4.0.0:
   dependencies:
     lodash._reinterpolate "^3.0.0"
 
+lodash.throttle@^4.1.1:
```

Comment:
I think you should run `yarn install` again to update this file since it still includes the caret